src/test/java/com/quirko/logic/LevelOperationsTest.java

```diff
@@ -0,0 +1,166 @@
+package com.quirko.logic;
```

Comment:
Bu dosyanın adı LevelTest.java olsun. LevelOperations diye bir class yok.

Reply:
Class and file name are changed.